### PR TITLE
feat: Give back enqueued value

### PR DESCRIPTION
**Pull request by jakoschiko**

### CODE REVIEW

File: src/server.rs [hunk 1]
```diff
@@ -202,6 +216,8 @@ pub enum ServerFlowEvent {
     ResponseSent {
         /// The handle of the enqueued [`Response`].
         handle: ServerFlowResponseHandle,
+        /// [`Response`] that was enqueued and is now sent.
```
Comment (duesee) [inline src/server.rs:219]: Maybe ...

```suggestion
        /// Formerly enqueued [`Response`] that was now sent.
```

**Review by duesee — APPROVED** [review]
Thanks! Looks good to me!

Surely, things got more complicated but luckily nothing of this is visible to our consumers.

Is there anything left you want to discuss? :-) For me it clearly looks like an improvement :+1:

File: proxy/src/proxy.rs [hunk 2]
```diff
@@ -309,26 +313,31 @@ fn handle_server_event(
...
             handle: _handle,
-            tag,
+            command,
             status,
         } => {
             // TODO: log handle
-            trace!(role = "p2s", ?tag, ?status, "---> Aborted command");
-            let status = match status.code() {
+            trace!(role = "p2s", ?command, ?status, "---> Aborted command");
+            let status: Status<'static> = match status.code() {
```
[the remark this answers is not in the document]
Comment (jakoschiko) [reply proxy/src/proxy.rs:328]: I committed it accidentally.

**Comment by jakoschiko** [comment]
> Is there anything left you want to discuss?

Nope. I resolved all conversions. Feel free to merge this if you're happy. Thanks.

**Pull request merged**